src/mender-update/daemon/states.cpp

```diff
@@ -176,6 +176,15 @@ void PollForDeploymentState::OnEnter(Context &ctx, sm::EventPoster<StateEvent> &
 		[&ctx, &poster](mender::update::deployments::CheckUpdatesAPIResponse response) {
 			if (!response) {
 				log::Error("Error while polling for deployment: " + response.error().String());
+
```

Comment:
double "*which*" in the commit message

Reply:
I wanted to type _with which_ something something